terraform/accounts/backups/main.tf

```diff
@@ -33,3 +33,23 @@ terraform {
   }
 }
 
+resource "aws_s3_bucket_policy" "backups_tfstate_bucket_policy" {
+  bucket = "digitalmarketplace-terraform-state-backups"
+  policy = jsonencode({
+    Version = "2012-10-17"
```

Comment:
Though, what's the significance of this date?

Reply:
It means we're using the latest version of the policy language and we have to include it - see https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html